decidim-core/app/forms/decidim/account_form.rb

```diff
@@ -60,10 +60,9 @@ def unique_email
     end
 
     def unique_nickname
-      return true if Decidim::User.where(
-        organization: context.current_organization,
-        nickname: nickname
-      ).where.not(id: context.current_user.id).empty?
+      return true if Decidim::User.where("decidim_organization_id = ? AND LOWER(nickname) = ? ", context.current_organization.id, nickname.downcase)
```

Comment:
Same here isn't it redundant ? 

Also, maybe a refactor is possible to use `current_organization` rather than `context.current_organization` since it is slower ? 

What do you think Armand ?